### DISCUSSION TITLE
Mystery 'Places' Dependancy in Storage target

### DISCUSSION THREAD

**Discussion by gutley** [mozilla-mobile/firefox-ios#12983] [General]
On browsing the various dependancies of all the targets, there is one in the Storage target called 'Places', but I can't find any relevant thing that builds it. Looks like its some kind of orphaned setting.

<img width="452" alt="image" src="https://user-images.githubusercontent.com/2040690/213450320-2d73b2ec-6766-46e1-a9f2-fec8a6c50263.png">

I removed it from my fork and nothing broke.

**Comment by OrlaM** [comment]
Looks like it was added in v98 as part of a fix to get SPM to play nice with M1 macs. It did nothing then either so it looks like it was just an artifact that slipped through when all the dependencies were being moved around to get things to work. I'll remove it from main shortly.

**Reply by lmarceau** [reply]
PR is up https://github.com/mozilla-mobile/firefox-ios/pull/13052

**Comment by ivan-pixel-rgb** [comment]
but the issue didn't solve anyone here to find the solution I also try on this sometimes working smoothly and sometimes disappear to do work on [SKM FILE](https:///www.hyena.com) to save us
